operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md

```diff
@@ -0,0 +1,25 @@
+# Add A Blade To A Cabinet In SLS Using CANI
+
+Begin by [initializing a session](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_init/), which will import data from SLS and create metadata in SLS' `ExtraProperties` field allowing the process to continue.
+
+```text
+ncn# cani alpha session init csm \
```

Comment:
Need to adhere to: https://github.com/Cray-HPE/docs-csm/blob/release/1.6/introduction/documentation_conventions.md#command-prompt-conventions

Reply:
Fixed